test/utils/Extensions.sol

```diff
@@ -214,6 +216,152 @@ contract LendingPoolExtension is LendingPool {
     }
 }
 
+contract LendingPoolExtension_NEW is LendingPool_NEW {
```

Comment:
Why do we need the full interface here?

Reply:
This is added more for the later step, when we add changes to the LendingPool_NEW that is related to liquidations, it will be used

Reply:
Just add the functions when you need them, otherwise it becomes impossible to maintain contracts.

Reply:
Deleted the functions that are not used, however its still necessary since the other extension is inheriting the old contract. The new test will not work.